index.js

```diff
@@ -65,10 +65,6 @@ function addNumericSeparator(num, str) {
     return $replace.call(str, sepRegex, '$&_');
 }
 
-var utilInspect = require('./util.inspect');
```

Comment:
This is incorrect; the util module is always present, and the browser field handles that for working bundlers.